full-moon/src/tokenizer.rs

```diff
@@ -587,8 +587,8 @@ impl<'ast> VisitMut<'ast> for TokenReference<'ast> {
 #[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
 pub struct Position {
     pub(crate) bytes: usize,
-    pub(crate) character: usize,
     pub(crate) line: usize,
+    pub(crate) character: usize,
```

Comment:
I added this change so that serialization is easier to read (having column then line number is confusing). And in this PR, so we don't have churn of all the `.snap` files.